src/query/src/sql.rs

```diff
@@ -229,6 +230,26 @@ pub async fn show_tables(
     }
 }
 
+pub fn show_variable(stmt: ShowVariables, query_ctx: QueryContextRef) -> Result<Output> {
+    let variable = stmt.variable.to_string().to_uppercase();
+    let value = match variable.as_str() {
+        "SYSTEM_TIME_ZONE" => get_timezone(None).to_string(),
```

Comment:
It's better to support `TIMEZONE` too. We support it in `set` statement.

Reply:
Do we need support `SYSTEM_TIMEZONE`? Or `TIMEZONE` just be fine

Reply:
Supports both are better.